MiniSim/Extensions/UserDefaults+Configuration.swift

```diff
@@ -15,6 +15,7 @@ extension UserDefaults {
         static let isOnboardingFinished = "isOnboardingFinished"
         static let enableiOSSimulators = "enableiOSSimulators"
         static let enableAndroidEmulators = "enableAndroidEmulators"
+        static let menuImage = "menu_icon_1"
```

Comment:
Let's create a new file `MenuIcons.swift` 

which would contain enum with all possible options: 

```swift
enum MenuImage: String {
    case iphone = "menu_icon_1"
    case ipad = "menu_icon_2"
    case box = "menu_icon_3"
  }
```

Reply:
Also can you rename the assets from menu icon to their actual descriptive name?